v3/spring-cloud-aws-v3-test/src/test/java/io/awspring/cloud/test/v3/cloudwatch/CloudwatchMetricsIntegrationTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.awspring.cloud.test.v3.cloudwatch;
+
+import java.time.Duration;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.containers.localstack.LocalStackContainer.Service;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.given;
+
+@SpringBootTest(properties = { "management.metrics.export.cloudwatch.namespace=awspring/spring-cloud-aws" })
+@Testcontainers(disabledWithoutDocker = true)
+class CloudwatchMetricsIntegrationTest {
```

Comment:
For Parameter Store and Secrets Manager, integration tests (those that use localstack) are just placed in the modules related to this integration. 

If we would have integration tests like before, that hit real AWS i somewhat see a sense of having a separate module, but in this case, shouldn't it just go to `metrics` module?

Reply:
I am following sqs in the `spring-cloud-aws-sqs` module. I think we should better discuss about how to handle this.